omod/src/main/webapp/fragments/dashboardwidgets/obsByEncounter.gsp

```diff
@@ -42,15 +42,28 @@
                     <td>
                         <span class="encounter-date">
                             <a class="visit-link" href="${ ui.urlBind("/" + contextPath + visitUrl, [ "patient.uuid": patient.id, "visit.uuid": enc.visit.uuid ]) }">
-                                ${ui.formatDatePretty(enc.encounterDatetime)}
+                                <% if (showTime) { %>
+                                    ${ui.formatDatetimePretty(enc.encounterDatetime)}
+                                <% } else { %>
+                                    ${ui.formatDatePretty(enc.encounterDatetime)}
+                                <% } %>
                             </a>
                         </span>
                     </td>
                     <td>
                         <span>
                             <% obsList.eachWithIndex { obs, index -> %>
-                                ${ui.format(obs)}${obsList.size() - 1 > index ? ", " : ""}
-                            <%  }%>
+                                <% if (obs.valueNumeric) { %>
+                                    <%  if ((minValue && obs.valueNumeric && (new BigDecimal(obs.valueNumeric) < new BigDecimal(minValue)))
```

Comment:
You don't need an extra obs.valueNumeric check here, as it is wrapped in one.

Reply:
Thanks! I have updated it.